digits/pretrained_model/views.py

```diff
@@ -3,17 +3,23 @@
 import tempfile
 import tarfile
 import zipfile
-import json
+
 
 import os
 import shutil
 import h5py
+import json
```

Comment:
If you're looking to clean up the imports, this is the order they should be in (without the comments):
```py
# standard imports
import json
import os
import shutil
import tempfile
import tarfile
import zipfile

# 3rd party
import flask
import h5py
import numpy as np
import werkzeug.exceptions

# this project
from digits import dataset, extensions, model, utils
from digits.inference import WeightsJob
from digits.pretrained_model import PretrainedModelJob
from digits.utils.routing import request_wants_json, job_from_request
from digits.views import get_job_list
from digits.webapp import app, scheduler
```
You don't actually have to clean them up everything here, but moving `json` down is a step in the wrong direction.